pivy-importer/src/main/groovy/com/linkedin/python/importer/deps/SdistDownloader.groovy

```diff
@@ -26,25 +26,23 @@ class SdistDownloader extends DependencyDownloader {
     static final String SOURCE_DIST_PACKAGE_TYPE = "sdist"
     static final String SOURCE_DIST_ORG = "pypi"
 
-    DependencySubstitution dependencySubstitution
-    boolean latestVersions
-    boolean allowPreReleases
-
     SdistDownloader(String project, File ivyRepoRoot, DependencySubstitution dependencySubstitution,
```

Comment:
Maybe you could use @InheritConstructors on this class: http://docs.groovy-lang.org/2.4.9/html/gapi/groovy/transform/InheritConstructors.html

Reply:
Good point!